static/src/stylesheets/module/_most-popular.scss

```diff
@@ -0,0 +1,295 @@
+            transform: translateY(-4px);
+            transition: transform .3s ease-in-out;
+        }
+
+        &:hover:after {
+            transform: translateY(0);
+        }
+
+        &:nth-child(2) {
+            &:before {
+                content: '';
+                width: 1px;
+                background: $brightness-86;
+                position: absolute;
+                left: 0;
+                top: 0;
+                bottom: 18px;
+            }
+        }
+    }
+
+    .tabs__tab--most-popular a {
+        font-weight: 700;
+        line-height: 1.1;
+        background-color: transparent;
+        box-sizing: border-box;
+        color: $brightness-7;
+        text-transform: capitalize;
+        height: 48px;
+        padding: ($gs-baseline / 2) ($gs-gutter / 2) 0;
+        display: block;
+        width: auto;
+
+        &:hover,
+        &:focus {
+            text-decoration: none;
+        }
+    }
+
+    .tabs__tab--selected {
+        background-color: $brightness-100;
+
+        &:after {
+            background-color: $brightness-46;
+            transform: translateY(0);
+        }
+    }
+}
+
+.most-popular__second-tier {
+    margin-top: $gs-baseline * 2;
+
+    @include mq(tablet) {
+        border-top: 1px solid $brightness-86;
+        display: flex !important;
+    }
+}
+
+.second-tier__item {
+    @include mq($until: tablet) {
+        border-top: 1px solid $brightness-86;
+    }
+
+    @include mq(tablet) {
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+        position: relative;
+
+        &:last-child {
+            &:before {
+                content: '';
+                position: absolute;
+                left: 0;
+                top: 0;
+                bottom: 0;
+                border-left: 1px solid $brightness-86;
+                z-index: 1;
+            }
+        }
+    }
+}
+
+// Simple content card
+.simple-content-card {
+    background-color: transparent !important;
+    color: $brightness-7;
+    padding: 0 ($gs-gutter / 2) $gs-baseline;
+    position: relative;
+    display: flex;
+
+    &:hover,
+    &:focus {
+        color: $brightness-7;
+        text-decoration: none;
+        background-color: $brightness-97 !important;
+
+        .fc-item__byline {
+            text-decoration: none;
+        }
+    }
+}
+
+.second-tier__item {
+    // Pseudo elements so the entire card is clickable
+    &:nth-child(1) .simple-content-card__headline:before,
+    &:nth-child(2) .simple-content-card__headline:before {
+        content: 'Most commented';
+        display: block;
+        font-weight: 700;
+        padding: ($gs-gutter / 6) 0 0;
+    }
+
+    &:nth-child(2) .simple-content-card__headline:before {
+        content: 'Most shared';
+    }
+}
+
+.simple-content-card__headline {
+    display: block;
+
+    .fc-item__byline {
+        display: block;
+    }
+}
+
+.simple-content-card__headline .js-headline-text {
```

Comment:
With the class `.js-headline-text` we use the `js-` prefix to indicate this class is a JavaScript selector. 
Ideally, classnames should be used for styling or JS, not both.

Reply:
I've added that in the cleanup story (https://trello.com/c/qETL0JDD/225-get-rid-of-the-old-most-pop-version) . Thanks Kate!